todo_app/tests/test_app.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pytest
+import requests
+from todo_app import app
+from dotenv import load_dotenv, find_dotenv
+from todo_app.data.config import TRELLO_BOARD_ID
+from todo_app.debug import debug
+from flask import current_app, request ##################???????????????
+
+# file_path = find_dotenv('.env.test')
+# load_dotenv(file_path, override=True)
+
+@pytest.fixture
+def client():
+    with current_app.app_context(): ##################???????????????
```

Comment:
I see various errors running this file, but I'm sure they are errors I saw you fix! Have you potentially missed pushing the latest changes to some of your code?